src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -62,4 +65,112 @@ public static function assertDateIntervalEquals(\DateInterval $expected, \DateIn
     {
         self::assertEquals($expected->format('%RP%yY%mM%dDT%hH%iM%sS'), $actual->format('%RP%yY%mM%dDT%hH%iM%sS'));
     }
+
+    public function testTextTypePlaceholderOption()
```

Comment:
The class `TypeTestCase` is extended by several test classes; consequently test methods are run for each child class. That's waste of machine time.

The assertion for each form type should be in the related test class (`TypeTestCase::testTextTypePlaceholderOption` become `TextTypeTest::testPlaceholderOption`)